tests/health_checks/NDv4/test-nccl.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+#Load the mpi module.
+module load mpi/hpcx
+
+#Lock graphics clocks to 1400.
+sudo nvidia-smi -lgc 1400 > /dev/null
```

Comment:
Since we are only running the very large size test we don't really need to lock the graphics clock. However, when running smaller size benchmarks it seems like the time for the GPU to spin up can make a significant difference. I've seen that locking the clocks improves performance on the smaller size nccl test runs (at least the ones using ib), and makes their performance more consistent.

Reply:
I see, thanks for your explanation :) 
I would add this as the comment, so others know the idea behind it.

Reply:
Great! I've added some explanation in the comments near this call.